scripts/reborn_webui_v2_live_qa/run_live_qa.py

```
@@ -4059,70 +4687,1709 @@ async def run_gated(ctx: LiveQaContext) -> ProbeResult:
+        if not status.get("ok"):
+            raise AssertionError(
+                "probe precondition failed: users.info could not read the "
+                f"QA account's status: {status.get('error')!r}"
+            )
+        status_text = str(status.get("status_text") or "").strip()
+        details["status_text"] = status_text
+        if not status_text:
+            raise AssertionError(
+                "probe precondition failed: no status is set on the QA account "
+                "— restore the OOO canary status fixture"
+            )
```

Ruff (0.15.20)

[warning] Abstract `raise` to an inner function

(TRY301)